README.md

```diff
@@ -223,7 +206,7 @@ Install the required packages by running the following command in the terminal
 Step 4:
 Run the following command in the terminal
 
-    python application.py
+    export FLASK_APP=FLASK_APP=fitnessapp && export FLASK_ENV=development && FLASK_DEBUG=1 && flask run
```

Comment:
"FLASK_APP=" repeated

Reply:
will correct that.

Reply:
I have corrected it in my branch

Reply:
preparing the pull request for event creation